### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Mikunj**
This PR adds tests for:
- `js/models/profile.js`
- `js/blocked_number_controller.js`

Might add more tests to this pr later.

### CODE REVIEW

File: test/models/profile_test.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        },
+      };
+      await storage.put(PROFILE_ID, profile);
+      assert.exists(storage.getLocalProfile().name);
+
+      await storage.setProfileName('');
+      assert.notExists(storage.getLocalProfile().name);
+    });
+  });
+});
```
Comment (BeaudanBrown) [inline test/models/profile_test.js]:
```suggestion
});

```

File: test/models/profile_test.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    it('trims the display name', async () => {
+      const values = [
+        {
+          current: '  prefix',
+          expected: 'prefix',
+        },
+        {
+          current: 'suffix  ',
+          expected: 'suffix',
+        },
```
Comment (sachaaaaa) [inline test/models/profile_test.js]: add a fixture for both prefix and suffix ?
```js
{
  current: ' both ',
  expected: 'both',
}
```

Comment (Mikunj) [reply test/models/profile_test.js]: would it be easier just testing 1 string instead?

```
    in   middle  
```

Comment (BeaudanBrown) [reply test/models/profile_test.js]: Ye I recon fine  to just do the one

**Pull request merged**